string group_name
int32 student_number
string[] student_names